linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/HttpConfig.scala

```diff
@@ -31,6 +31,7 @@ class HttpInitializer extends ProtocolInitializer.Simple {
       .prepend(Headers.Dst.PathFilter.module)
       .replace(StackClient.Role.prepFactory, DelayedRelease.module)
       .prepend(http.ErrorResponder.module)
+      .insertAfter(http.ErrorResponder.role, ResponseFramingFilter.module)
```

Comment:
@olix0r, Alex & I discussed this, and we weren't sure if this ought to go above or below the failure accrual module in the client stack. What do you think?